PushNotifications/PushNotifications/Instance.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+struct Instance {
+    static func persist(_ instanceId: String) throws {
+        guard let savedInstanceId = Instance.getInstanceId() else {
+            UserDefaults(suiteName: "PushNotifications")?.set(instanceId, forKey: "com.pusher.sdk.instanceId")
```

Comment:
I'd make this a constant somewhere so that you can't change this and not the one in `getInstanceId`